### PR TITLE
Feature/452772 query email send bug fix

### PR DESCRIPTION
**Pull request by johnfitzgeraldEviden**
Added Welsh email templates and updated code to handle Welsh version of emails.

### CODE REVIEW

File: src/EPR.RegulatorService.Facade.API/appsettings.json [hunk 1]
```diff
@@ -83,8 +83,8 @@
     "AccountCreationUrl": "https://localhost:7854/create-account/invitation/",
     "OrganisationRegistrationSubmissionQueriedId": "cda3cad1-fa77-4282-b39b-7524b232615d",
```
Comment (karthikeviden) [inline src/EPR.RegulatorService.Facade.API/appsettings.json:84]: shouldn't all these be in secrets?

File: src/EPR.RegulatorService.Facade.Core/Models/Accounts/EmailModels/OrganisationRegistrationSubmissionEmailModel.cs [hunk 2]
```diff
@@ -33,10 +33,21 @@ public Dictionary<string, object> GetParameters {
                     { "organisation_number", this.OrganisationNumber },
                     { "organisation_name", this.OrganisationName },
                     { "agency", this.Agency },
-                    { "agency_email", this.AgencyEmail },
                     { "year", this.Period }
                 };
 
+                if (IsWelsh.Value)
+                {
+                    var emails = this.AgencyEmail.Split(';');
```
Comment (karthikeviden) [inline src/EPR.RegulatorService.Facade.Core/Models/Accounts/EmailModels/OrganisationRegistrationSubmissionEmailModel.cs:41]: better to have a null check before picking the items from array?

Comment (johnfitzgeraldEviden) [reply src/EPR.RegulatorService.Facade.Core/Models/Accounts/EmailModels/OrganisationRegistrationSubmissionEmailModel.cs:41]: Value will only return the value if it has been assigned to a nullable type.  So no need for null check.  Unless I am mistaken

File: src/EPR.RegulatorService.Facade.API/appsettings.json (hunk 1 above)
Comment (johnfitzgeraldEviden) [reply src/EPR.RegulatorService.Facade.API/appsettings.json:84]: All the other email ID are stored within the app setting file.

**Review by karthikeviden — APPROVED** [review]

**Review by shehzad-ismail8 — APPROVED** [review]

**Review by JulianYoungEviden — APPROVED** [review]

**Pull request merged**